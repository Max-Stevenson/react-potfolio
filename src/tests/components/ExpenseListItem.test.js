import ExpenseListItem from '../../components/ExpenseListItem';
import { shallow } from 'enzyme';
import React from 'react';
import expenses from '../fixtures/expenses';

test('Should render ExpenseListItem with expenses', () => {
  const expense = expenses[1];
  const wrapper = shallow(<ExpenseListItem {...expense}/>);
  expect(wrapper).toMatchSnapshot();
});