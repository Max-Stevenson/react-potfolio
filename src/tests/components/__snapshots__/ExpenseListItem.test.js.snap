// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseListItem with expenses 1`] = `
<Link
  className="list-item"
  to="/edit/2"
>
  <div>
    <h3
      className="list-item__title"
    >
      test two
    </h3>
    <span
      className="list-item__subtitle"
    >
      December 28th, 1969
    </span>
  </div>
  <h3
    className="list-item__data"
  >
    Â£0.01
  </h3>
</Link>
`;
